src/containers/Wallet.tsx

```diff
@@ -0,0 +1,41 @@
+import { Wallet as WalletComponent } from '../components/Wallet/Wallet';
+import { useEffect, useRef, useState } from 'react';
+import { useGetAccountsLazyQuery } from '../hooks/accounts/queries/useGetAccountsQuery';
+import { useGetExtensionQuery } from '../hooks/extension/queries/useGetExtensionQuery';
+import { useSetActiveAccountMutation } from '../hooks/accounts/mutations/useSetActiveAccountMutation';
+import { useGetActiveAccountQuery } from '../hooks/accounts/queries/useGetActiveAccountQuery';
+
+export const Wallet = () => {
+  const { data: extensionData, loading: extensionLoading } =
+    useGetExtensionQuery();
+  const [setActiveAccount] = useSetActiveAccountMutation();
+  const { data: activeAccountData } = useGetActiveAccountQuery();
+  const [getAccounts, { data: accountsData }] = useGetAccountsLazyQuery();
```

Comment:
you can also use a normal query (not lazy) and use the `skip: boolean` option to defer it's execution. In this case it's be `{ skip: !isAccountSelectorOpen }`. But take this with a pinch of salt, but i think you should be able to achieve the same in this case as with the lazy query, with less code (no manual effects)

Reply:
implemented